README.md

```diff
@@ -37,17 +39,8 @@ $ aws-vault exec home -- aws s3 ls
 bucket_1
 bucket_2
 
-# store credentials for the "work" profile
-$ aws-vault add work
-Enter Access Key Id: ABDCDEFDASDASF
-Enter Secret Key: %
-
-# Execute a command using temporary credentials
-$ aws-vault exec work -- aws s3 ls
-another_bucket
-
 # Inspect the environment
-$ aws-vault exec work -- env | grep AWS
+$ aws-vault exec home -- env | grep AWS
 AWS_VAULT=work
```

Comment:
Hopefully if you look at the new version it makes sense. Basically I am trying to tighten up the README here by not explicitly showing off multiple profiles.